spot-ml/src/main/scala/org/apache/spot/proxy/ProxyWordCreation.scala

```diff
@@ -56,18 +52,22 @@ object ProxyWordCreation {
                 responseCode: String,
                 topDomains: Broadcast[Set[String]],
                 agentCounts: Broadcast[Map[String, Long]],
-                timeCuts: Array[Double],
-                entropyCuts: Array[Double],
-                agentCuts: Array[Double]): String = {
+                entropyCuts: Array[Double]): String = {
     Try{
       List(topDomain(proxyHost, topDomains.value).toString,
-        Quantiles.bin(TimeUtilities.getTimeAsDouble(time), timeCuts).toString,
+        // Time binned by hours
+        TimeUtilities.getTimeAsHour(time).toString,
         reqMethod,
+        // Fixed and equally spaced cutoffs
```

Comment:
there's a risk that this comment will rot if the cutoff values are changed... just strike "and equally spaced"